packages/perseus-editor/src/widgets/graded-group-editor.tsx

```diff
@@ -33,16 +33,17 @@ class GradedGroupEditor extends React.Component<Props> {
         hint: null,
     };
 
+    editor = React.createRef<Editor>();
+    hintEditor = React.createRef<Editor>();
```

Comment:
They're both `Editor`? `hintEditor` doesn't need to be something like `HintEditor`? I imagine there's some shared interface or inheritance or [some other important sounding type word]?

Reply:
Yes, in this case both refs are an `Editor`. I think it's because a Hint in Perseus is very, very close to a regular question. 

<img width="227" alt="image" src="https://github.com/Khan/perseus/assets/77138/a4a42207-43b4-4895-9a47-84bb61761833">